apps/src/music/player/MusicPlayer.ts

```diff
@@ -518,56 +519,44 @@ export default class MusicPlayer {
 
       return results;
     } else if (event.type === 'chord') {
-      const chordEvent = event as ChordEvent;
-      const {instrument, notes, playStyle} = chordEvent.value;
-      if (notes.length === 0) {
-        return [];
-      }
+      const results: SampleEvent[] = this.convertChordEventToSampleEvents(event);
+      return results;
+    }
 
-      const results: SampleEvent[] = [];
-
-      if (playStyle === 'arpeggio-up') {
-        notes.sort();
-      } else if (playStyle === 'arpeggio-down') {
-        notes.sort().reverse();
-      } else if (playStyle === 'arpeggio-random') {
-        // Randomize using Fisher-Yates Algorithm
-        for (let i = notes.length - 1; i > 0; i--) {
-          const j = Math.floor(Math.random() * (i + 1));
-          const temp = notes[i];
-          notes[i] = notes[j];
-          notes[j] = temp;
-        }
-      }
+    return [];
+  }
 
-      // Create the array of samples. If the play style is arpeggio, we play one
-      // sound every 16th note, repeating the sequence as many times as needed.
-      // If the play style is "together", then only add each note once, and have them
-      // all play at the start of the given measure.
-      for (let i = 0; i < (playStyle === 'together' ? notes.length : 16); i++) {
-        const note = notes[i % notes.length];
-        const sampleId = this.getSampleForNote(note, instrument);
-        if (sampleId === null) {
-          console.warn(
-            `No sound for note value ${note} on instrument ${instrument}`
-          );
-          continue;
-        }
+  convertChordEventToSampleEvents(event: PlaybackEvent) : SampleEvent[] {
```

Comment:
nit: formatting

Reply:
Thanks.  Looking forward to `prettier` working all the time!